pypika/dialects.py

```diff
@@ -36,6 +45,22 @@ def __init__(self, **kwargs: Any) -> None:
         super().__init__(dialect=Dialects.SNOWFLAKE, **kwargs)
 
 
+class SnowflakeCreateQueryBuilder(CreateQueryBuilder):
+    QUOTE_CHAR = None
```

Comment:
I noticed CreateQueryBuilder specifies ALIAS_QUOTE_CHAR=None but SnowflakeQueryBuilder specifies '"'. I don't think this will cause any issues but noting this here anyway.